server/utils/courseParticipationUtils.ts

```diff
@@ -159,4 +141,28 @@ export default class CourseParticipationUtils {
       value: { text: valueTextItems.join(', ') || 'Not known' },
     }
   }
+
+  // eslint-disable-next-line @typescript-eslint/member-ordering
```

Comment:
Do you have a strong preference for having these methods listed in display order rather than the default alphabetical?

Reply:
I'd like them to be below `private static summaryListRows(`, if only we could have `private private` methods!

Reply:
What if they were `summaryListRowsOutcome` etc instead of `outcomeSummaryListRow`, then we'd get it for free and also get the sorting check within the set of methods? It's more namespaced that way, which I think is the motivation for this sorting (?), but maybe less plain language